data/json/professions.json

```diff
@@ -1526,12 +1528,12 @@
     "type": "profession",
     "ident": "bionic_customer",
     "name": "Commercial Cyborg",
-    "description": "You always had to have the latest and best gadgets and gizmos, so is it any wonder that you upgraded your flesh along with your smart phone?  Only time will tell if your status as a marvel of bionic technology will be enough to ensure your survival after the apocalypse.",
-    "points": 6,
+    "description": "You always had to have the latest and best gadgets and gizmos, so is it any wonder that you upgraded your flesh along with your smart phone?  Only time will tell if your passion for electronics and your status as a marvel of bionic technology will be enough to ensure your survival after the apocalypse.",
+    "points": 8,
     "CBMs": [ "bio_flashlight", "bio_tools", "bio_ups", "bio_watch", "bio_batteries", "bio_power_storage_mkII" ],
-    "skills": [ { "level": 1, "name": "electronics" } ],
+    "skills": [ { "level": 4, "name": "electronics" }, { "level": 2, "name": "fabrication" } ],
```

Comment:
I'm don't it makes a lot of sense for a consumer end cyborg to be an 8 point build with high starting skills. This is the equivalent to the person who always has the latest iPhone, not a tech guru.

Reply:
Reasoning is that they arr really into the latest tech so they would know about it. For example, how to install a ups mod to their electronics to be able to use it from their bionic's power source.